lib/order.rb

```diff
@@ -0,0 +1,32 @@
+class Order
+  attr_reader :selected_items
+  def initialize(menu_class = Menu, dish_class = Dish, confirmation_class = Confirmation)
+    @selected_items = []
+    @menu = menu_class.new([dish_class.new("chips", 1.50), dish_class.new("nuggets", 2.00),
+    dish_class.new("big mac", 3.50), dish_class.new("apple pie", 0.75)])
```

Comment:
Could you store this data somewhere else? The contents of this menu shouldn't be stored in the Order class.

Reply:
Does an order need to instantiate it's own menu? It might be an idea to pass in a menu instance here, instead of the menu class. Same goes for Dishes.